src/components/EventModal/Collect/index.vue

```diff
@@ -22,6 +22,21 @@
       v-if="isCompleted"
       #message
     >
+      <client-only>
+        <model-viewer
```

Comment:
should add a margin top for this element

Reply:
how much do you suggest?

Reply:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/6198816/216653548-e57c97b5-eda9-41d1-bf77-74e6f7f845e2.png">
effect of 12px

Reply:
Seems ok